mne/inverse_sparse/mxne_optim.py

```diff
@@ -471,19 +471,22 @@ def mixed_norm_solver(M, G, alpha, maxit=3000, tol=1e-8, verbose=None,
     dgap_freq : int
         The duality gap is computed every dgap_freq iterations of the solver on
         the active set.
-    active_set_init : array or None
-        The initial active set used at the first iteration. If None, the
-        usual active set strategy is applied.
-    X_init : array or None
+    active_set_init : array, shape (active_set_size,) or None
```

Comment:
it's not a boolean mask as done elsewhere for active_set variable?

Reply:
Indeed, it is a boolean mask. I've said it line 475 ("The initial active set (boolean array) used at..."). Do you want me to put it elsewhere?

Reply:
then the shape is wrong. It should be n_dipoles no?